### PR TITLE
ci: Run all tests, on a periodic run by schedule

### PR DESCRIPTION
**Pull request by abetomo**
Time-expensive tests are skipped on CI, but this may delay noticing failures.
Change to run all tests once a day.
It usually takes about 2 hours, so we set a timeout of 150 minutes.

### CODE REVIEW

File: test/test-pgroonga-check.rb [hunk 1]
```diff
@@ -167,7 +167,7 @@ class PGroongaCheckTestCase < Test::Unit::TestCase
         input.puts(<<-"SQL")
 SELECT path
 FROM (
-  SELECT value->1 name, value->2 path
+  SELECT value->1 AS name, value->2 AS path
```
Comment (abetomo) [inline test/test-pgroonga-check.rb:170]: Add `AS`, since it fails in PostgreSQL 12 and 13.

**Review by otegami — APPROVED** [review]
LGTM 👍🏾

File: test/test-pgroonga-check.rb [hunk 2]
```diff
@@ -151,7 +151,7 @@ class PGroongaCheckTestCase < Test::Unit::TestCase
 
   sub_test_case "list-broken-indexes" do
     test "corrupt" do
-      omit("This test takes over 10 minutes, so skip it.") if ENV["CI"]
+      omit("This test takes over 10 minutes, so skip it.") if ENV["CI"] == "true"
```
Comment (kou) [inline test/test-pgroonga-check.rb:154]:
```suggestion
      omit("This test takes over 10 minutes.") if ENV["CI"] == "true"
```

**Comment by kou** [comment]
Let's try.

**Pull request merged**